squad/frontend/user_settings.py

```diff
@@ -55,8 +57,42 @@ def api_token(request):
     return render(request, 'squad/user_settings/api_token.html', context)
 
 
+@login_required
+def subscriptions(request):
+
+    subscriptions = Subscription.objects.filter(user=request.user)
+    groups = Group.objects.all().prefetch_related('projects')
+
+    if request.method == "POST":
+        for project_id in request.POST.getlist("subscriptions[]"):
+            try:
+                Subscription.objects.create(
+                    project_id=project_id,
+                    user=request.user)
+            except IntegrityError:
+                #log that object was not added.
+                pass
+
+        return redirect(reverse('settings-subscriptions'))
+
+    context = {
+        'subscriptions': subscriptions,
+        'groups': groups
+    }
+    return render(request, 'squad/user_settings/subscriptions.html', context)
+
+
+@login_required
+def remove_subscription(request, id):
+
+    Subscription.objects.get(id=id).delete()
```

Comment:
you need to filter on the current user before deleting, oltherwise this opens a breach where a user can remove subscriptions from another user.